src/test/java/org/opensearch/security/dlic/rest/api/legacy/LegacySslCertsApiTest.java

```diff
@@ -1,29 +0,0 @@
-/*
```

Comment:
qq, why delete this test?

Reply:
All API calls now are randomly selected between old style and a new one here: 
https://github.com/opensearch-project/security/blob/0bfe7653b0f50ff6b6757411af4e49d9784a063b/src/integrationTest/java/org/opensearch/security/api/AbstractApiIntegrationTest.java#L248

Reply:
Correct